modules/ROOT/pages/variable-transformer-reference.adoc

```diff
@@ -16,7 +16,7 @@ for complex scenarios.
 
 | Variable Name (`variableName`)
 | Required
-| Name of the variable. Names can only include numbers, characters, and underscores. For example, hyphens are not allowed in the name.
+| Name of the variable, which can be a string or a DataWeave expression. Variable names can only include numbers, characters, and underscores. For example, hyphens are not allowed in the name.
```

Comment:
```suggestion
| Name of the variable, which can be a string or a DataWeave expression. Variable names can include only numbers, characters, and underscores. For example, hyphens are not allowed in the name.
```

Reply:
@ca-stefan-cordes Can you commit this edit. Then we'll re-approve and merge. Thanks for your contribution.